.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 4.8.4
+- 8.9.0
```

Comment:
why not just `- 8` since we're doing effectively that in the `package.json` file?